.github/workflows/develop.yml

```diff
@@ -1,58 +1,31 @@
-name: Build BTS and Run Build and Test in Container
+name: Push nightly BTS Image
 
 on:
-  pull_request:
-    branches: [develop, master]
-
-env:
-  DOCKER_BUILDKIT: 1
-  COMPOSE_DOCKER_CLI_BUILD: 1
+  push:
+    branches:
+      - develop
 
 jobs:
-  small_test_bts:
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Build images BeaverTripleService
-        run: make test m=build
-        working-directory: ./Test/
-
-      - name: Run test in BeaverTripleService
-        run: make test m=run
-        working-directory: ./Test
-
-  small_test:
-    runs-on: ubuntu-latest
-    needs: [small_test_bts]
-
-    steps:
-      - name: Merge All small_test
-        run: echo "ok"
-
-  # LargeテストのためCCのイメージをpush
   build_and_push_bts:
     runs-on: ubuntu-latest
 
     steps:
       - name: checkout
         uses: actions/checkout@v2
 
-      - name: Build CC image
+      - name: Build BeaverTripleService images
         run: |
-          docker build ../../ --file Dockerfile --target dev --tag ghcr.io/${{ github.repository_owner }}/bts_dev:${{ github.run_id }}
-          docker build ../../ --file Dockerfile --target medium --tag ghcr.io/${{ github.repository_owner }}/bts_medium:${{ github.run_id }}
+          docker build ../../ --file Dockerfile --target dev --tag ghcr.io/${{ github.repository_owner }}/quickmpc-bts:${{ github.run_id }}
+          docker build ../../ --file Dockerfile --target dep-runner --tag ghcr.io/${{ github.repository_owner }}/quickmpc-bts:${{ github.run_id }}-alpine
```

Comment:
Unlike before, the stage name and image name are different, so I need a comment in the readme or dockerfile.

Reply:
@mdonaka Noted with thanks.
By 2aabc5b23e0de11abd54bb12da643291364bba4f, documentation about docker image and build stage was written in README.md at top directory of repository.

Reply:
I checked it. Very good!